src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+
+    private double radius;
+
+    public Circle(double radius, String color) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public double getRadius() {
+        return this.radius;
+    }
+
+    public void setRadius(double radius) {
+        this.radius = radius;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }

```

Comment:
Please move getters/setters from all child classes to parent class where field is stored